Source/WebInspectorUI/UserInterface/Models/RecordingAction.js

```diff
@@ -115,6 +115,39 @@ WI.RecordingAction = class RecordingAction extends WI.Object
         return typeof propertyDescriptor.value === "function";
     }
 
+    static bitfieldNamesForParameter(type, name, value, index, count)
+    {
+        if (value === 0)
+            return null;
+
+        function test_and_clear_bit(names, value, bit, desc) {
+            if (value & bit)
+                names.push(desc);
+            return value & ~bit;
+        }
+
+        let prototype = WI.RecordingAction._prototypeForType(type);
+        let names = [];
+
+        if ((name === "clear" && index === 0 && (type === WI.Recording.Type.CanvasWebGL || type === WI.Recording.Type.CanvasWebGL2)) ||
+            (name === "blitFramebuffer" && index === 8 && type === WI.Recording.Type.CanvasWebGL2)) {
+            value = test_and_clear_bit(names, value, prototype.COLOR_BUFFER_BIT, "COLOR_BUFFER_BIT");
+            value = test_and_clear_bit(names, value, prototype.DEPTH_BUFFER_BIT, "DEPTH_BUFFER_BIT");
+            value = test_and_clear_bit(names, value, prototype.STENCIL_BUFFER_BIT, "STENCIL_BUFFER_BIT");
+            if (value !== 0)
+                names.push(value.toString(16));
+        }
+
+        if (name === "clientWaitSync" && index === 1 && type === WI.Recording.Type.CanvasWebGL2) {
+            value = test_and_clear_bit(names, value, prototype.SYNC_FLUSH_COMMANDS_BIT, "SYNC_FLUSH_COMMANDS_BIT");            if (value !== 0)
```

Comment:
oops i think you may have missed a newline 😅

Reply:
Oops.